src/components/pages/Theme/Theme.jsx

```diff
@@ -261,6 +262,42 @@ export const Theme = () => {
         </Segment>
       </div>
 
+      <div className='section'>
+        <Segment padded>
+          <Header as='h2' dividing>
+            Chip
+          </Header>
+          <div className='row'>
```

Comment:
Let's first have a `.row`. of examples that are not interactive and do not have an icon. 

Then we can use the `<Header>` and `.section` class for "Outlined" and "Interactive" sections, like we do the "Disabled" and "Form" checkboxes.